src/constants.js

```diff
@@ -123,4 +61,16 @@ export const FLEX_ITEM_LIST = [
   'alignSelf'
 ]
 
+export const COMMON = composeList(COMMON_LIST)
+export const BORDER = composeList(BORDER_LIST)
+export const TYPOGRAPHY = composeList(TYPOGRAPHY_LIST)
+export const LAYOUT = composeList(LAYOUT_LIST)
+export const POSITION = composeList(POSITION_LIST)
+export const FLEX_CONTAINER = composeList(FLEX_CONTAINER_LIST)
+export const FLEX_ITEM = composeList(FLEX_ITEM_LIST)
+
+function composeList(list) {
+  return compose(...list.map(name => styles[name]))
```

Comment:
I'm not 100% sure what the best name for this function and its argument is 🤷‍♂️

Reply:
I think this is pretty good naming as is!